config/application.rb

```diff
@@ -92,6 +92,9 @@ class Application < Rails::Application
     # The time(in minutes) until the `load_min_user_count` will be used for calculating server load
     config.x.load_join_buffer_time = ENV.fetch('LOAD_JOIN_BUFFER_TIME', 15).to_i.minutes
 
+    # Whether to generate ids for servers based on the hostname rather than random UUIDs
+    config.x.server_id_is_hostname = ENV['SERVER_ID_IS_HOSTNAME']&.downcase == 'true'
```

Comment:
We should add a default value and in order to keep consistency at this point we should favour ENV.fetch

`config.x.recording_disabled = ENV.fetch('SERVER_ID_IS_HOSTNAME', 'false').casecmp?('true')`